brave-core/src/main/java/com/github/kristofa/brave/Brave.java

```diff
@@ -257,17 +257,22 @@ public AnnotationSubmitter serverSpanAnnotationSubmitter() {
     }
 
     private Brave(Builder builder) {
-        SpanFactory spanFactory = builder.spanFactoryBuilder.build();
```

Comment:
changes here yank the old stuff out (while still keeping deprecated code working). was very hard work, but portability is important!